content/v2.0/account-management/multi-user/invite-user.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Invite a user
+list_title: Invite a user
+description: >
+  Invite a user to collaborate
+weight: 103
+menu:
+  v2_0:
+    parent: Manage multiple users
+    name: Invite a user
+---
+
+## Invite user from {{< cloud-name >}}
```

Comment:
drop line 13 to eliminate the double heading for now...when we have  multiple Cloud/OSS sections, we can add intro sentence and new headers